hummingbot/client/config/config_validators.py

```diff
@@ -1,19 +1,17 @@
 from hummingbot.core.utils.trading_pair_fetcher import TradingPairFetcher
-from hummingbot.client.settings import (
-    EXCHANGES,
-    STRATEGIES,
-)
 from decimal import Decimal
 from typing import Optional
 
 
 # Validators
 def validate_exchange(value: str) -> Optional[str]:
+    from hummingbot.client.settings import EXCHANGES
```

Comment:
This import and the below import are to avoid circular import as well?

Reply:
Yes.